modules/cco-ccoctl-configuring.adoc

```diff
@@ -201,6 +201,14 @@ Ensure that the architecture of the `$RELEASE_IMAGE` matches the architecture of
 ====
 
 . Extract the `ccoctl` binary from the CCO container image within the {product-title} release image by running the following command:
+//Below ifeval added to reflect OSDOCS-12321
+ifeval::["{context}" == "installing-gcp-customizations"]
+[IMPORTANT]
++
+====
+Find the identity pool ID and the provider ID. During cluster installation, the names of these resources is provided to `ccoctl` with the `--name parameter`.
```

Comment:
Change to: "For GCP Workload Identity, you need to provide the identity pool ID and provider ID to ccoctl with the --name parameter.  Record these values for later, which you will need to  xref:../../storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc#installing-the-gcp-filestore-csi-driver-operator[install the GCP Filestore Container Storage Interface (CSI) Driver Operator]."

Not sure if I got the number of levels up right for the xref, so check this.